<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPromote</class>
 <widget class="QDialog" name="DialogPromote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>135</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_rook">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>20</y>
     <width>77</width>
     <height>92</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_knight">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>20</y>
     <width>87</width>
     <height>92</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_bishop">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>86</width>
     <height>92</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_queen">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>92</width>
     <height>92</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
